renderer/components/Home/WalletUnlocker.js

```diff
@@ -82,15 +82,15 @@ class WalletUnlocker extends React.Component {
           <>
             <WalletHeader wallet={wallet} />
 
-            <Input
+            <PasswordInput
+              autoComplete="current-password"
               field="password"
-              id="password"
               isRequired
               label={<FormattedMessage {...messages.wallet_unlocker_password_label} />}
               minLength={8}
               my={3}
+              name="password"
```

Comment:
You can actually remove the name prop. `Input` component should set the `name` prop to the same value as `field`

Reply:
I should ignore this comment as per Slack, right? Same for the below comments.

Reply:
No, please remove. I created https://github.com/LN-Zap/zap-desktop/pull/2767 to address the root issue

Reply:
remove name="password" everywhere where we set field="password"